appinventor/components/src/com/google/appinventor/components/runtime/Clock.java

```diff
@@ -605,6 +611,31 @@ public static String FormatTime(Calendar instant) {
     return Dates.FormatTime(instant);
   }
 
+  /**
+   * Changes the timezone of a given instant.
+   *
+   * @param instant  instant to modify
+   * @param timezone  timezone to apply to instant
+   * @return  updated instant
+   */
+  @SimpleFunction(description = "Updates the timezone in which the instant is saved at")
+  public static Calendar ChangeTimezone(Calendar instant, String timezone) {
+    if (timezones.contains(timezone))
```

Comment:
Suggestion: trim spaces and ignore case in comparison

Reply:
![image](https://user-images.githubusercontent.com/19251112/72627149-43710700-394c-11ea-95b6-ea17075f37df.png)

TimeZone is actually case sensitive, so "america/detroit" will not be a valid timezone. We should not allow users to input it lowercase even though we know it is "America/Detroit", as it's not officially supported by Java.

Reply:
It is case-sensitive, that's why I mentioned:
> If found, then use the actual value that is in the list (not the passed-in parameter) and set the timezone to the found-value